vlib/v/checker/tests/expr_evaluated_but_not_used.vv

```diff
@@ -0,0 +1,4 @@
+fn main() {
+	{1, 2, 3, 4}
+	println('works')
+}
```

Comment:
This may be a better test input, since it will make the error point the last expression statement in the block:
```v
fn main() {
    {
        println(1)
        println(2)
        println(3)
        println(4)
        5
    }
    println('works')
}
```